### DISCUSSION TITLE
How to get the identification of a user to fetch data?

### DISCUSSION THREAD

**Discussion by pedromsl** [nextauthjs/next-auth#1636] [Help]
Hello,
I'm a bit confused with this whole authentication process. I managed to login successfully and get access_token, etc, in the server the signing was successful. However, I'm confused.
How would I go about making the client get the data for their specific account? The client, from what I've seen, only gets the email, name, and photo (at least from Google). So, how can the client fetch the data with some sort of identifier? How and what do I save in the DB (I'm using MongoDB)

**Comment by balazsorban44** [comment]
https://next-auth.js.org/configuration/callbacks#jwt-callback

https://next-auth.js.org/configuration/callbacks#session-callback
